tmt-app/src/Components/InfoLinkPage.jsx

```diff
@@ -26,41 +28,72 @@ function MajorPage() {
       name: "BobaTalks Discord Server",
       link: "https://discord.com/invite/bobatalks",
     },
+    ...(params.career
+      ? [
+          {
+            name: "Career Information via Zippia",
+            link: data.link,
+          },
+        ]
+      : []),
   ];
-  let sectionInfo = [
-    {
-      title: `What is ${
-        ["a", "e", "i", "o", "u"].includes(data.name[0].toLowerCase())
-          ? "an "
-          : "a "
-      }
+  let sectionInfo = params.major
+    ? [
+        {
+          title: `What is ${
+            ["a", "e", "i", "o", "u"].includes(data.name[0].toLowerCase())
```

Comment:
Let's pull this out and make it into a util

Reply:
how would we do this?

Reply:
The concept of abstraction is really just strategic placement of functions. Whenever you find logic is getting complex or very specific, you can encapsulate it in a function and call the function to give you your desired output.

Consider SOLID principles as well too--even if not word for word, following some of this makes for cleaner code

Reply:
I wasn't sure how to do this since it's in an array of objects, but I think I might have an idea now.